src/client/src/ui/spotTile/components/PriceButton.tsx

```diff
@@ -34,17 +34,17 @@ const Box = styled('div')`
 `
 
 const DirectionLabel = styled(Box)`
-  color: ${({ theme: { text } }) => text.textMeta};
+  opacity: 0.59;
```

Comment:
Does `.59` do the trick? 😛

No request for a change — I just always question the relative value of being particular vs a more regular choice. I will admit though, I have been inclined to `.6` at times when `.5` or `.75` didn't cut it 😅

More importantly though, this is a good time to go and try out `green.D4` or `green.D3` and see if it's not more striking.

Opaque colors chosen from our palette should always be preferred to an opacity. A platform cannot be trusted to mix colors consistently.

Reply:
This is just an opacity that matches the design's opacity.

Reply:
You'll see this across some of the other components. It would be great to have an added function to handle this.